src/worker.js

```diff
@@ -286,14 +286,7 @@ self.onmessage = (e) => {
     } else if (e.data.target === 'setimmediate') {
       // no-op
     } else if (e.data.cmd === 'processProxyingQueue') {
-      try {
-        if (Module['_pthread_self']()) { // If this thread is actually running?
-          Module['_emscripten_proxy_execute_queue'](e.data.queue);
-        }
-      } finally {
-        // Always decrement the ref count.
-        Atomics.sub(HEAP32, e.data.queue >> 2, 1);
-      }
+      executeNotifiedProxyingQueue(e.data.queue);
```

Comment:
I don't think you can call library functions directly from `worker.js` can you?   I think you would need to make sure the library function was exported on the module and then call `Module[' executeNotifiedProxyingQueue']`.   Ensure its exported and not minimized is kind of tricky... and happens in couple of different places I think (for example MINIMAL_RUNTIME has explicit code to export these on the Module).